Bottom: 632eb6bce63f2a90b556684bdcdaf10b0b883847
Top:    5f1e36014715b8d5401c0e7021b6924e060fddaa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-02 18:12:35 +0000

Refresh of fixup-frac_circle

---

diff --git a/src/polygon.c b/src/polygon.c
index e724649..bbd8e7a 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -320,7 +320,7 @@ frac_circle (PLINE * c, LocationType X, LocationType Y, Vector v, int range)
   e1 = v[0] - X;
   e2 = v[1] - Y;
 
-  range = (range == 1) ? CIRC_SEGS-1 : (CIRC_SEGS / range);
+  range = (range == 1) ? CIRC_SEGS : (CIRC_SEGS / range) - 1;
   for (i = 0; i < range; i++)
     {
       /* rotate the vector */
